frontend/src/components/Pagination.js

```diff
@@ -6,28 +6,25 @@ import { ButtonGroup, Button } from "react-bootstrap";
 const Pagination = (props) => {
   let renderPaginationBox = (content, pageIndex) => (
     <Button
-      key={`pagination-box-${pageIndex}`}
+      key={`pagination-box-${content}`}
       className="PaginationBox"
-      variant="secondary"
+      variant={(pageIndex === props.currentPageIndex ? "primary" : "secondary")}
       onClick={() =>
         pageIndex !== props.currentPageIndex && props.onSetPage(pageIndex)
       }
     >
-      {pageIndex !== props.currentPageIndex && <a>{content}</a>}
-      {pageIndex === props.currentPageIndex && content}
+    {content === pageIndex ? pageIndex + 1 : content}
     </Button>
   );
 
-  let pagesToRender = props.pageIndexEnd - props.pageIndexStart;
+  let pagesToRender = props.numberOfPages;
   if (!pagesToRender) return <ul />;
   return (
     <ButtonGroup>
       {props.currentPageIndex !== 0 && renderPaginationBox("First", 0)}
+      {props.currentPageIndex !== 0 && renderPaginationBox("Prev", 0)}
       {_.times(pagesToRender, (i) =>
-        renderPaginationBox(
-          props.pageIndexStart + i + 1,
-          i + props.pageIndexStart
-        )
+        renderPaginationBox(i, i)
```

Comment:
Why do you pass 'i' twice here?

Reply:
The first i stands for the content of the pagination button while the second stands for the page index of that button, in some case we will use it like `renderPaginationBox("Next", pageIndex)`.